tests/golden.csv

```diff
@@ -3611,6 +3947,9 @@ Shashank Vineet,yournews.com,2020-06-02,https://yournews.com/2020/04/21/1578589/
 Pete,youtube.com,2020-06-17,https://www.youtube.com/,FALSE,,,,
 skip,youtube.com,2020-06-17,https://www.youtube.com/watch?v=T7NcQ8lHHLc,TRUE,en-us,2020-03-04,,#Coronavirus Doctors and nurses in Iran are lifting the morale of COVID-19 patients through dance.
 Selina Ho,za.linkedin.com,2020-06-30,https://za.linkedin.com/,FALSE,,,,
+Shashank Vineet,zaralawny.com,2020-07-06,https://www.zaralawny.com/,FALSE,,,,
+Shashank Vineet,zaralawny.com,2020-07-06,https://www.zaralawny.com/french/,FALSE,,,,
+Shashank Vineet,zaralawny.com,2020-07-06,https://www.zaralawny.com/french/articles/pdf/2010-12-02%20reviewed%20FDA%20Francais-SC.pdf,TRUE,zh,2010-12-02 00:00:00,,
```

Comment:
I think this line is not correct since it's a pdf file.  Is that right?

Reply:
Yes, it is. Sorry about that, I can change that.